content/installation/net/install/Supported-technology-net.md

```diff
@@ -32,5 +32,17 @@ The .NET agent for Azure App Service supports analysis of web applications built
 | Processor Architecture     | Both 32-bit and 64-bit applications are supported. |
 | Web Application Frameworks | All of the frameworks supported by the Windows agent are analyzed on Azure. |
 
-
 > **Note:** Only applications using the full .NET Framework are currently supported. Contrast plans to add support for .NET Core applications in the future.
+
+## .NET for Docker
+
+The .NET agent can run in docker on a Windows based image that .NET Framework 4.5.1 and IIS.  We recommend basing your image on [microsoft/aspnet](https://hub.docker.com/_/microsoft-dotnet-framework-aspnet).  Please see [.NET Docker documentation](http://todo.com) for detailed installation instructions.
```

Comment:
todo.com?

Reply:
fixed but kaitlyn would need to set the final link

Reply:
Put the examples in a separate repo and linked to them.